Feat/wp roles (#250)
okp4/docs: docs/whitepaper/roles.md

***Entity***
Validators typically operate as entities or individuals running network nodes.

:::note

To maintain balance and security in the network, a limit has been set on the number of Validators: no more than 100 Validators can operate at any given time.
To maintain the integrity of our network, any inappropriate or malicious behavior from Validators is met with penalties. This disciplinary action serves to uphold the network's standards and security, keeping it robust and reliable for all participants.
More information about [Tendermint BFT mechanism](https://tendermint.com/core/)

:::

### Delegators

***Purpose***
Delegators primarily delegate their $KNOW tokens and voting power to Validators to accumulate rewards.

***Responsibilities***
They are responsible for choosing whom to delegate their tokens and voting power to. Furthermore, they need to be aware of the risks associated with delegating their rights.

***Interaction***

> Delegators engage with the protocol by delegating their tokens to Validators. However, they can manually “retrieve” their voting rights and directly participate in governance through specific transaction

Reviewer: ```suggestion
Delegators engage with the protocol by delegating their tokens to Validators. However, they can manually “retrieve” their voting rights and directly participate in governance through specific transaction.
```

***Entity***
Delegators are typically individuals or entities staking $KNOW tokens.

### DAO Participants

***Purpose***
The principal role of DAO Participants is to participate in the on-chain governance of the protocol. They have the power to propose and vote on protocol improvement propositions.

***Responsibilities***
DAO Participants are responsible for issuing improvement proposals, voting on these proposals, and staking $KNOW tokens to participate in governance. As protocol holders, they also have a direct stake in its direction and success.

***Interaction***
DAO Participants interact with the protocol by making and voting on proposals, facilitated by staking $KNOW. Each $KNOW token provides a voting right.

***Entity***
DAO Participants can only be delegators or Validators for voting. Nevertheless, note that it is necessary to deposit $KNOW to make a proposal and therefore not necessarily need delegation

## Dataverse & Zone Participants

### Data Providers

***Purpose***
Data Providers primarily aim to reference datasets within the protocol.

***Responsibilities***

> They are responsible  describing dataset characteristics (metadata) and for establishing consent rules for the use of this resource . They are solely authorized to modify these rules unless they decide otherwise through these rules of consent. They may also receive compensation from Consumers each time their data is used or purchased, according to the associated governance rules.

Reviewer: ```suggestion
They are responsible  describing dataset characteristics (metadata) and for establishing consent rules for the use of this resource . They are the only ones authorized to modify these rules unless they decide otherwise through these rules of consent. They may also receive compensation from Consumers each time their data is used or purchased, according to the associated governance rules.
```

***Interaction***

> They interact with the protocol via dedicated smart contracts to register datasets, establish their consent rules and characteristics, and make the dataset available and accessible. In addition, retribution fees may be distributed to data providers, in accordance with established consent rules.

Reviewer: ```suggestion
They interact with the protocol via dedicated smart contracts to register datasets, establish their consent rules and characteristics, and make the dataset available and accessible. In addition, retribution fees may be distributed to data providers, in accordance with established consent rules and zone rules.
```

***Entity***
Data Providers can be individuals, applications, companies, or organizations owning and/or managing datasets.

### Service Providers

***Purpose***
Service Providers mainly aim to provide resources such as algorithms, software, storage systems, or any other digital resource requiring processing time.

***Responsibilities***

> Service Providers are responsible for providing services, defining consent rules, describing services, and setting up remuneration models based on the nature and usage of their resources. depending on consent rules and Zone rules, service providers may be required to stake tokens to ensure service availability and integrity.

Reviewer: ```suggestion
Service Providers are responsible for providing services, defining consent rules, describing services, and setting up remuneration models based on the nature and usage of their resources. Depending on consent rules and Zone rules, service providers may be required to stake tokens to ensure service availability and integrity.
```

***Interaction***

> Service Providers interact with the protocol by referencing their services and consent rules via smart contracts, ensuring the availability and functioning of their service in accordance with on-chain registered specifications. Furthermore, retribution fees may be distributed to service providers, according with established consent rules.

Reviewer: ```suggestion
Service Providers interact with the protocol by referencing their services and consent rules via smart contracts, ensuring the availability and functioning of their service in accordance with on-chain registered specifications. Furthermore, retribution fees may be distributed to service providers, according with established consent rules and zone rules.
```
